doc/release-notes-22539.md

```diff
@@ -0,0 +1,8 @@
+Notable changes
+===============
+
+P2P and network changes
+-----------------------
+
+- Fee estimation now takes the feerate of replacement (RBF) transactions into
```

Comment:
I have a question about the desired behavior because it's a subtle thing that isn't mentioned here: with this PR, if a transaction is RBF'ed in our mempool, we add the replacement transaction and remove the original transaction(s) from the fee estimator. If the original transactions end up being mined in a block anyway (e.g. a miner had already started working on it before the replacement transaction was accepted), we lose that data in fee estimation. Is this what we want? If yes, maybe document?

Reply:
> Is this what we want?

Yes, because there is nothing we can do about that. And such a race sounds to be very improbable.. (are there still mining pools refusing RBFs for whatever reason?)

>  If yes, maybe document?

This is not a change in behaviour, i don't think it should be in the changelog. Before this patch if you accept a replacement transaction in your mempool but the original tx ends up being mined you would lose the data anyways.

Reply:
Ah you're right, we're already removing the original transactions. My mistake!